sample/shared/compose/src/commonMain/kotlin/com/arkivanov/sample/shared/root/RootContent.kt

```diff
@@ -109,6 +125,10 @@ fun RootContent(component: RootComponent, modifier: Modifier = Modifier) {
                 label = { Text(text = "Custom Nav", softWrap = false) },
             )
         }
+
+        dialogComponent?.let {
```

Comment:
While we are here, let's use `also` instead of `let`, because `let` is has mapping meaning. Also the `dialogComponent` variable can be inlined in this case, because it's used only once: `dialogState.overlay?.instance?.also {`.

Reply:
Fixed, thanks for pointing out